transit_odp/fares/extract.py

```diff
@@ -188,6 +188,14 @@ def _get_count(self, attr):
         lens = [len(lines) for lines in self._attr_from_documents(attr)]
         return sum(lens)
 
+    def _get_user_type_count(self, attr):
+        """Gets the total count of distinct UserType in all documents"""
+        lists_user_types = self._attr_from_documents(attr)
+        distinct_user_types = set(
+            [value for sublist in lists_user_types for value in sublist]
```

Comment:
You should probably convert each value into lower case and then compare?

Reply:
This should not be required as xml schema validation will fail if there is any uppercase value in it..

lxml.etree.XMLSyntaxError: Element '{http://www.netex.org.uk/netex}UserType': [facet 'enumeration'] The value 'Senior' is not an element of the set {'adult', 'child', 'infant', 'senior', 'student', 'youngPerson', 'schoolPupil', 'military', 'disabled', 'disabledCompanion', 'jobSeeker', 'employee', 'animal', 'anyone'}.

Reply:
Ohh makes sense